index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Reviews Slider</title>
+    <!-- styles -->
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/11a1e92bc5.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <p class="title">Our Reviews</p>
+        <hr />
+        <article class="review">
+          <img src="person-1.jpeg" id="person-img" alt="" />
```

Comment:
@Saminsfw What if use an image from api as you did in the reviewsObject and then delete the image altogether?

Reply:
Done

Reply:
I can still see the image, can you please remove that?